api/app/signals/settings/base.py

```diff
@@ -378,7 +378,7 @@
     'SECURITY_DEFINITIONS': {
         'Signals API - Swagger': {
             'type': 'oauth2',
-            'authorizationUrl': '{url}oauth2/authorize'.format(url=DATAPUNT_API_URL),
+            'authorizationUrl': f'{DATAPUNT_API_URL}oauth2/authorize',
```

Comment:
Consider adding a note to `DATAPUNT_API_URL = os.getenv('DATAPUNT_API_URL', 'https://api.data.amsterdam.nl/')` that it must be terminated by a slash.